subcommands/ptar/ptar.1

```diff
@@ -0,0 +1,76 @@
+.Dd June 26, 2025
+.Dt PLAKAR-PTAR 1
+.Os
+.Sh NAME
+.Nm plakar ptar
+.Nd generate a self-contained Kloset archive (.ptar)
+.Sh SYNOPSIS
+.Nm
+.Op Fl plaintext
+.Op Fl overwrite
+.Op Fl k | kloset Ar location
+.Fl o Ar file.ptar
+.Op Ar path ...
+.Sh DESCRIPTION
+The
+.Nm
+command creates a single portable archive (a
+.Pa .ptar
```

Comment:
`Pa` is for whole paths (e.g. `/etc/resolv.conf`), i'd just use `Em` to require an italic or just `Dq`/`Sq` for quoting)